### PR TITLE
Nexus common

### PR DESCRIPTION
**Pull request by jdillon**

### CODE REVIEW

File: components/nexus-common/src/main/java/org/sonatype/nexus/common/io/DirSupport.java [hunk 1]
```diff
@@ -481,7 +479,7 @@ public static void applyToFiles(final Path from, final Function<Path, FileVisitR
   private static void validateDirectory(final Path... paths) {
     for (Path path : paths) {
       checkNotNull(path, "Path must be non-null");
-      checkArgument(Files.isDirectory(path), SimpleFormat.template("%s is not a directory", path));
+      Preconditions.checkArgument(Files.isDirectory(path), String.format("%s is not a directory", path));
```
Comment (adreghiciu) [inline components/nexus-common/src/main/java/org/sonatype/nexus/common/io/DirSupport.java]: static import?

File: components/nexus-common/src/main/java/org/sonatype/nexus/common/io/FileSupport.java [hunk 2]
```diff
@@ -153,7 +151,7 @@ public static void writeFile(final Path file, final Charset charset, final Strin
   private static void validateFile(final Path... paths) {
     for (Path path : paths) {
       checkNotNull(path, "Path must be non-null");
-      checkArgument(Files.isRegularFile(path), SimpleFormat.template("%s is not a regular file", path));
+      Preconditions.checkArgument(Files.isRegularFile(path), String.format("%s is not a regular file", path));
```
Comment (adreghiciu) [inline components/nexus-common/src/main/java/org/sonatype/nexus/common/io/FileSupport.java]: Static import

File: components/nexus-common/src/main/java/org/sonatype/nexus/common/io/DirSupport.java (hunk 1 above)
Comment (jdillon) [inline components/nexus-common/src/main/java/org/sonatype/nexus/common/io/DirSupport.java]: yup will tidy this up

**Comment by jdillon** [comment]
http://bamboo.s/browse/NX-OSSF472-4 :white_check_mark:

**Comment by jdillon** [comment]
manually merged

**Pull request closed without merging**